### DISCUSSION TITLE
Add conditions/rules to customize request details

### DISCUSSION THREAD

**Discussion by enz1ey** [sct/overseerr#3118] [Feature Requests]
I am currently using Petio and honestly will continue using it until Overseer allows us more control over requests than simply approving or not approving. A huge feature Petio offers is being able to set up rules in order to customize various aspects of a request. For instance, I keep a separate path for any animated TV shows so they're kept in a separate library than my "normal" TV shows. Not necessarily "anime" but just anything animated. So I can create a rule where if the genre contains "animation" then the root folder is different than other Sonarr requests.

Also, I can change the quality profile based on the release year of a show/movie. For instance, my default Radarr profile is 720p/1080p, but I have a rule set up in Petio for any movies older than 2005, the quality profile is set to "any" instead of the default profile.

![Screenshot 2022-11-15 at 2 57 03 PM](https://user-images.githubusercontent.com/6290727/202014369-86b0b812-a91f-4992-a5a3-e973bd8db0aa.png)

**Comment by BionicAlice** [comment]
This would be perfect. Its why I restrict tv shows to approval so i can link them to the correct root folder. I have mine split up by animation/documentary/normal/reality. Would be nice to have it on auto approve as I don't notice a request right away and takes me time at times.

**Comment by jtrumpower** [comment]
I too would love this.

It would be very helpful if I could configure movies/shows to automatically go to certain root folders based on their rating or genre. For example, an R rated show would default to (and possibly be unchangeable) a specific root folder. The same could be done with certain genres like Horror.

Having many different users for my content, it is helpful to have different libraries for different age groups. Giving people request permission often results in them requesting something and putting it in the wrong root folder. Resulting in Horror or R rated films landing in a "family" library.

**Comment by tvann2182** [comment]
I was just getting ready to add this to a discussion and decided to search and found this. I used to use ombi, petio and now overseer and this is a feature that is definitely wanted!

**Comment by lakshyaGoel** [comment]
This would be really beneficial. I have two instances of radar—one for English and one for foreign languages. The same is true for sonar. Adding a filter would make it easy for me to direct requests to the correct instance.

**Comment by Skepmeister** [comment]
would love this functionality to be added.  More options the better.

**Comment by DamienDessagne** [comment]
This is what prevents me from moving to Overseer. In Petio, I can redirect french movies/shows to a special profile in Radarr/Sonarr, same goes with animes, etc. This is definitely a must have feature.
